Ftp/New-FtpDirectory/New-FtpDirectory.psm1

```diff
@@ -129,18 +129,18 @@ function New-FtpDirectory
                     try
                     {
                         $webclient.UploadFile($uri, $srcFullPath)
-                        Write-Verbose "Upload successful."
+                        Write-Verbose 'Upload successful.'
 
                         break
                     }
                     catch
                     {
                         try
                         {
-                            Write-Warning "Error caught, trying initializing new webclient object."
+                            Write-Warning 'Error caught, trying initializing new webclient object.'
 
                             $errorCount++
-                            Write-Debug "ERROR COUNT:" $errorCount
+                            Write-Debug 'ERROR COUNT:' $errorCount
```

Comment:
Apparently this was bugged.

![image](https://user-images.githubusercontent.com/4007293/213820169-d1aff3a5-723c-4a73-b672-b0113f10b859.png)


```suggestion
                            Write-Debug "ERROR COUNT: $errorCount"
```